test/Microsoft.Azure.WebJobs.Host.FunctionalTests/Queues/QueueTests.cs

```diff
@@ -143,8 +143,9 @@ public class Poco
 
             // BindingData is case insensitive. 
             // And queue name is normalized to lowercase. 
+            // Connection="" is same as Connection=null
             public const string QueueOutName = "qName-{XYZ}";
-            public void Func([QueueTrigger(QueueName)] Poco triggers,  [Queue(QueueOutName)] ICollector<string> q)
+            public void Func([QueueTrigger(QueueName, Connection ="")] Poco triggers,  [Queue(QueueOutName)] ICollector<string> q)
```

Comment:
Rather than have the test hidden here, it would be better to have a simple targeted unit test on the GetStorageAccountAsync method itself. Having it hear doesn't make it clear that this is something we're trying to regression test.